src/mozilla_sec_eia/models/sec_eia_record_linkage/transform_eia_input.py

```diff
@@ -0,0 +1,107 @@
+"""Create an EIA input utilities table that's ready for record linkage with the SEC 10K companies."""
```

Comment:
This is the new module I made for creating the SEC output table asset. It does a lot (maybe too much):

* cleaning/preprocessing on the basic 10k and Ex. 21 assets to make them ready to be inputs to the output table
* matches the Ex. 21 subsidiaries to basic 10k filers if that subsidiary also files a 10k
* adds columns like parent_company_cik
* concatenates the 10K filers and Ex. 21 subsidiaries (who don't file a 10k themselves)